cluster/juju/layers/kubernetes-master/config.yaml

```diff
@@ -7,3 +7,7 @@ options:
     type: string
     default: cluster.local
     description: The local domain for cluster dns
+  service-cidr:
+    type: string
+    default: 10.152.183.0/24
+    description: CIDR to user for Kubernetes services. Cannot be changed after deployment.
```

Comment:
Acknowledging immutable config sadly does not make it go away. And we all know immutable config breaks user experience. If this value can not be changed... Can we consider it a layer option?